bundles/org.openhab.binding.doorbird/pom.xml

```diff
@@ -12,22 +12,24 @@
 
   <name>openHAB Add-ons :: Bundles :: Doorbird Binding</name>
 
-  <properties>
-    <dep.noembedding>jna</dep.noembedding>
-  </properties>
-
   <dependencies>
     <dependency>
       <groupId>com.goterl.lazycode</groupId>
       <artifactId>lazysodium-java</artifactId>
-      <version>4.0.1</version>
-      <scope>compile</scope>
+      <version>4.2.5</version>
+      <scope>provided</scope>
     </dependency>
+ 	<dependency>
+      <groupId>co.libly</groupId>
+      <artifactId>resource-loader</artifactId>
+      <version>1.3.7</version>
+      <scope>provided</scope>
+	</dependency>
     <dependency>
       <groupId>net.java.dev.jna</groupId>
       <artifactId>jna</artifactId>
-      <version>5.4.0</version>
-      <scope>compile</scope>
+      <version>5.5.0</version>
+      <scope>provided</scope>
     </dependency>
```

Comment:
are any of these even used in this binding?

Reply:
They wouldn't be there if the binding didn't use them.

Reply:
Ok, so that answer was a little flippant. 😉 

The Doorbird broadcasts doorbell press and motion detection events on a UDP port. Those UDP packets are encrypted. The lazysodium-java library is an OSGI container that wraps the libsodium native code crypto library. Lazysodium-java requires a NativeLong object, which is part of jna. It also uses resource-loader to load the libsodium.so (or libsodium.dll) library either from the OS, or from the libsodium libs bundled in the lazysodium-java jar file.

Reply:
That is fine, my question was flippant as well. I was having trouble finding the code that used these dependencies but you helped me fine them, so thanks. Looks like the makers of lazysodium-java don't know how to publish their dependencies properly on maven.

Reply:
> Looks like the makers

If you can help me understand what's wrong, I can ask the him to make the necessary changes. He's been very helpful in working through some issues, as well as making the latest release of the libs OSGI compliant.

Reply:
I may have rushed to judgement. My key observation about that project is that instead of including `resource-loader` as a maven dependency they included the raw jar file. Raw files can't be used as transitive dependencies.

Reply:
Ok. Is there anything to do here, or can we consider this resolved?